www/guides/create_account.md

```diff
@@ -0,0 +1,186 @@
+const AXproxyConstructorCallData = stark.compileCalldata({ 
+    implementation: argentXaccountClassHash, 
+    selector: hash.getSelectorFromName("initialize"), 
+    calldata: stark.compileCalldata({ signer: starkKeyPubAX, guardian: "0" }), 
+});
+const AXcontractAddress = hash.calculateContractAddressFromHash(
+    starkKeyPubAX, 
+    argentXproxyClassHash, 
+    AXproxyConstructorCallData, 
+    0
+);
+console.log('Precalculated account address=', AXcontractAddress);
+```
+If you want a specific private key, replace `stark.randomAddress()` by your choice.  
+Then you have to fund this address.  
+### deployment of the new account :
+If you have sent enough fund to this new address, you can go forward to the final step :
+```typescript
+const accountAX = new Account(provider, AXcontractAddress, starkKeyPairAX);
+const deployAccountPayload = { 
+    classHash: argentXproxyClassHash, 
+    constructorCalldata: AXproxyConstructorCallData, 
+    contractAddress: AXcontractAddress, 
+    addressSalt: starkKeyPubAX };
+const { transaction_hash: AXdAth, contract_address: AXcontractFinalAdress } = await accountAX.deployAccount(deployAccountPayload);
+console.log('✅ ArgentX wallet deployed at :',AXcontractFinalAdress);
+```
+## Create your account abstraction :
+You are not limited to OZ or AX contracts ; you can create your own contract for wallet. It's the concept of Account Abstraction.
+You can customize entirely the wallet ; for example :
+- use a different concept of keys.
+- Add a guardian to save your account.
+- have the possibility to transfer the ownership of the wallet.
+- Add some administrators or a super-administrator.
+- Whitelist of address for transfer.
+- Multisig  
+
+The only limitation is your imagination...  
+Here an example of a customized wallet, including super administrator management, on a local starknet-devnet (launch `starknet-devnet --seed 0`  before using this script) :
+```typescript
+import { Account, ec, json, stark, Provider, hash } from "starknet";
+import axios from "axios";
+```
+
+```typescript
+// connect provider
+const provider = new Provider({ sequencer: { network: "http://127.0.0.1:5050" } });
+
+// initialize existing predeployed account 0 of Devnet
+const privateKey0 = "0xe3e70682c2094cac629f6fbed82c07cd";
+const starkKeyPair0 = ec.getKeyPair(privateKey0);
+const accountAddress0 = "0x7e00d496e324876bbc8531f2d9a82bf154d1a04a50218ee74cdd372f75a551a";
+const account0 = new Account(provider, accountAddress0, starkKeyPair0);
+
+// new account abstraction :
+    // Generate public and private key pair.
+const AAprivateKey = stark.randomAddress();
+console.log('New account :\nprivateKey=', AAprivateKey);
+const AAstarkKeyPair = ec.getKeyPair(AAprivateKey);
+const AAstarkKeyPub = ec.getStarkKey(AAstarkKeyPair);
+console.log('publicKey=', AAstarkKeyPub);
+
+// declare the contract
+const compiledAAaccount = json.parse(fs.readFileSync("./compiled_contracts/myAccountAbstraction.json").toString("ascii")
```

Comment:
Could I ask you to add a section about clarifying this `json.parse` somewhere?
It created a confusion in this issue: https://github.com/0xs34n/starknet.js/issues/473#issuecomment-1371931040

Reply:
I already made a paragraph in connect_contract.md : https://github.com/PhilippeR26/starknet.js/blob/guidesPRO/www/guides/connect_contract.md#get-the-abi-from-a-compiled-file-
I will add something in create_account.md.

Reply:
Done.